core/src/types/operator/operator.rs

```diff
@@ -1201,6 +1201,26 @@ impl Operator {
     /// # }
     /// ```
     ///
+    /// ## `if_none_match`
+    ///
+    /// Set `if_none_match` for this `write` request.
+    ///
+    /// This feature can be used to check if the file already exists.
+    /// This prevents overwriting of existing objects with identical key names.
+    /// And must use the * (asterisk) value with this parameter
```

Comment:
S3 only supports `*`, but other services may support different options. We can revise the statement to say users can use `*` to verify if a file already exists by matching with any ETag.